pkg/relayer/proxy/metrics.go

```diff
@@ -3,10 +3,21 @@ package proxy
 import (
 	stdprometheus "github.com/prometheus/client_golang/prometheus"
 
-	"github.com/go-kit/kit/metrics"
 	"github.com/go-kit/kit/metrics/prometheus"
 )
 
+const (
+	relayMinerSubsystem = "relayminer"
```

Comment:
Thoughts on "Process" or "Server" instead of "Subsystem". No strong opinion from my end.